charts/nri-kube-events/Chart.yaml

```diff
@@ -10,10 +10,4 @@ keywords:
   - events
   - newrelic
   - monitoring
-maintainers:
-  - name: alvarocabanas
-  - name: carlossscastro
-  - name: gsanchezgavier
-  - name: kang-makes
-  - name: paologallinaharbur
-  - name: roobre
+deprecated: true
```

Comment:
Regarding all deprecated charts being bold we could get rid of the code as specified in https://helm.sh/docs/topics/charts/#deprecating-charts

Reply:
if we decide so I would open an issue to do that in a separate PR

Reply:
Maybe we should do that once we decide that no security update are going to be released

Reply:
It says that we have to release it so even if we want to remove the source code we would have to remove the chart in a follow-up PR.

On the other hand, I am against it (yet)